assets/components/ovn/master/daemonset.yaml

```diff
@@ -120,32 +120,26 @@ spec:
           trap quit TERM INT
 
           bracketify() { case "$1" in *:*) echo "[$1]" ;; *) echo "$1" ;; esac }
+          compact() {
+            while true; do
+              /usr/bin/ovn-appctl -t /var/run/ovn/ovn${1}_db.ctl --timeout=5 ovsdb-server/compact 2>/dev/null || true
+              sleep 600
+            done
+          }
           # initialize variables
           db="nb"
           ovn_db_file="/etc/ovn/ovn${db}_db.db"
-
           OVN_ARGS="--db-nb-cluster-local-port=9643 --no-monitor"
 
           echo "$(date -Iseconds) - starting nbdb"
-          initialize="false"
-
-          if [[ ! -e ${ovn_db_file} ]]; then
-            initialize="true"
-          fi
+          exec /usr/share/ovn/scripts/ovn-ctl \
+            ${OVN_ARGS} \
+            --ovn-nb-log="-vconsole:${OVN_LOG_LEVEL} -vfile:off -vPATTERN:console:%D{%Y-%m-%dT%H:%M:%S.###Z}|%05N|%c%T|%p|%m" \
+            run_nb_ovsdb &
 
-          if [[ "${initialize}" == "true" ]]; then
```

Comment:
I see this is a cleanup, since both paths of the if do the same.

Reply:
yes.